### ISSUE TITLE
The increment webopoly win logic is broken

### ISSUE THREAD

**Issue by timflannagan** [Daniel-Pivonka/timbot#84]
Whoever implemented the increment webopoly logic did not test this at all lol

Before incrementing a win, and running locally, here are the current standings:
```
dan is the reigning champ with 116 wins
tim is next with 35 wins
nathan is next with 3 wins
adam is next with 0 wins
And in last is loser alfredo with 0 wins
```

After incrementing a win for Adam, and then subsequently relisting the standings, his score jumped from 0 -> 8 wins:
```
dan is the reigning champ with 116 wins
tim is next with 35 wins
adam is next with 8 wins
nathan is next with 3 wins
And in last is loser alfredo with 0 wins
```

**Comment by timflannagan** [comment]
Or this might have been fixed recently but dan's local container version is out of date :shrug:

**Comment by nathan-weinberg** [comment]
Yeah this should be fixed by #83
